privacyidea/api/lib/prepolicy.py

```diff
@@ -988,10 +988,15 @@ def check_token_upload(request=None, action=None):
     :param filename:
     :return:
     """
-    upload_allowed = Match.admin(g, action=ACTION.IMPORT, user_obj=request.User).allowed()
-    if not upload_allowed:
-        raise PolicyError("Admin actions are defined, but you are not allowed"
-                          " to upload token files.")
+    tokenrealms = request.all_data.get("tokenrealms")
+    if tokenrealms:
```

Comment:
This breaks the test for the client IP in [`test_api_lib_policy.test_03_check_token_upload()`](https://github.com/privacyidea/privacyidea/blob/074cf3dcb7713456c50cdd49b991e68c41094d4d/tests/test_api_lib_policy.py#L290) which does not specify a `tokenrealm`.

Reply:
I fixed the logic in 48297db3